File: PYTHON/SUM OF ELEMENTS/sum.py
def sumelements(arr):

    sum = 0

    for i in arr:

        sum += i

    return sum

def sumelements2(arr):

    return sum(arr)


arr = [3,35,34,456,5765,756,7678,24,3434,5435]

print("Sum Of Elements = " , sumelements(arr))
print("Sum Of Elements = " , sumelements2(arr))

File: JAVA/SUM OF ELEMENTS/Sum.java
public class Sum {
    
    public static void main(String[] args) {
        
        int arr[] = {2,5,46,7,67,678,2,434,45,56};

        System.out.println("Total Elements Sum Is = " + sum_elements(arr));
    }

    public static int sum_elements(int arr[]){

        int sum = 0;

        for (int i : arr) {
            
            sum += i;

        }

        return sum;
    }
}
